### ISSUE TITLE
[Feature request] Different diametrizer for each neurite type

### ISSUE THREAD

**Issue by adrien-berchet** [BlueBrain/NeuroTS#87]
### Describe the feature

Currently we can only provide one diametrizer that is used for all neurite types (though a diametrizer can be configured differently for each neurite type). I think it could be interesting to be able to provide one different diametrizer for each neurite type. WDYT?

### Example Use

A `tmd_parameters.json` could have an entry like:
```JSON
"diameter_params": {
    "basal_dendrite": {
        "method": "uniform",
        "diameter": 0.6
    },
    "apical_dendrite": {
        "method": "default"
    },
    "axon": {
        "method": "M1"
    }
}
```

### Use cases

I don't know if such a use case is relevant for a user, I just realized that currently we can't synthesize morphologies with this kind of configurations.

**Comment by arnaudon** [comment]
we can do it if its needed, otherwise I wouldn't do it 'for free'

**Comment by adrien-berchet** [comment]
I agree, it's just an idea but let's wait to see if this use case actually arises.
